packages/core/src/messagingProtocols/mls/MLSService/CoreCryptoMLSError.ts

```diff
@@ -17,6 +17,21 @@
  *
  */
 
+const CORE_CRYPTO_ERROR_NAMES = {
+  ConversationAlreadyExists: 'ConversationAlreadyExists',
```

Comment:
You need to include `MlsError` prefix:

MlsErrorConversationAlreadyExists

Reply:
https://github.com/wireapp/wire-web-packages/pull/6749/commits/988f9907c370b1a2e4e3c392fdb0d9a5e0764c63